src/control/cmd/dmg/pretty/storage_test.go

```diff
@@ -1555,13 +1558,13 @@ host1
 host1
 -----
   Devices
-    UUID:00000000-0000-0000-0000-000000000000 [TrAddr:0000:8a:00.0 NSID:0]
+    UUID:00000000-0000-0000-0000-000000000000 [TrAddr:0000:8a:00.0 NSID:1]
       Roles:wal SysXS Targets:[0 1 2] Rank:0 State:NEW LED:OFF
-    UUID:00000001-0001-0001-0001-000000000001 [TrAddr:0000:8b:00.0 NSID:0]
+    UUID:00000001-0001-0001-0001-000000000001 [TrAddr:0000:8b:00.0 NSID:1]
       Roles:data,meta Targets:[3 4 5] Rank:0 State:EVICTED LED:ON
-    UUID:00000002-0002-0002-0002-000000000002 [TrAddr:0000:da:00.0 NSID:0]
+    UUID:00000002-0002-0002-0002-000000000002 [TrAddr:0000:da:00.0 NSID:1]
       Roles:wal SysXS Targets:[0 1 2] Rank:1 State:UNKNOWN LED:NA
-    UUID:00000003-0003-0003-0003-000000000003 [TrAddr:0000:db:00.0 NSID:0]
+    UUID:00000003-0003-0003-0003-000000000003 [TrAddr:0000:db:00.0]
```

Comment:
NIT.
From my understanding this case should never happen.
It could be more relevant to mock an unplugged device in this case ?

Reply:
This unit test case is verifying the translation of devices in response to display output and specifically what happens when you have a zero value NSID reported for some reason. I don't think it's a problem that it's not an exact real-world output.

Reply:
Fair enough.